x-pack/plugins/security_solution/server/endpoint/services/feature_usage/service.ts

```diff
@@ -14,6 +14,7 @@ const FEATURES = {
   HOST_ISOLATION_EXCEPTION_BY_POLICY: 'Host isolation exception by policy',
   TRUSTED_APP_BY_POLICY: 'Trusted app by policy',
   EVENT_FILTERS_BY_POLICY: 'Event filters by policy',
+  BLOCKLIST_BY_POLICY: 'Blocklists by policy',
```

Comment:
Good catch. We'll need to remind ourselves of ensuring we make this update with future artifacts